DIGNDB.App.SmitteStop.DAL/Repositories/TemporaryExposureKeyRepository.cs

```diff
@@ -30,15 +34,36 @@ public async Task AddTemporaryExposureKey(TemporaryExposureKey temporaryExposure
 
         public async Task AddUniqueTemporaryExposureKeys(IList<TemporaryExposureKey> temporaryExposureKeys)
```

Comment:
I suppose it would be worth our while to make this method async now it's stated to be - or remove async